gemfirexd/core/src/main/resources/com/pivotal/gemfirexd/internal/impl/sql/catalog/metadata_lc.properties

```diff
@@ -0,0 +1,1793 @@
+#
```

Comment:
In order to see which all metadata queries we have modified, compare this file with https://github.com/SnappyDataInc/snappy-store/blob/SDENT-170/gemfirexd/prebuild/src/main/resources/com/pivotal/gemfirexd/internal/impl/jdbc/metadata.properties